lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.test.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+
+/* eslint-disable max-len */
+/* eslint-disable no-unused-expressions */
```

Comment:
If in most test files we rely on `chai` then it might be convienient to turn it off for all test files in eslint config